test/java/src/com/ibm/streamsx/topology/test/api/ParallelTest.java

```diff
@@ -43,6 +43,85 @@
 import com.ibm.streamsx.topology.tuple.BeaconTuple;
 
 public class ParallelTest extends TestTopology {
+
+    @Test
+    public void testCatsCradle() throws Exception {
+	checkUdpSupported();
+	Topology topology = new Topology("testCat");
+	TStream<String> out = topology.strings("hello").parallel(5, TStream.Routing.HASH_PARTITIONED).filter(nopStringFilter()).endParallel()
```

Comment:
Is there a simple way to have tests that actually test the parallelism requested was implemented?

By just passing a single tuple through it seems they would pass if the implementation actually didn't use parallelism?

Maybe the functions being parallelized could actually test the max channel number?